identifiers/country-pa/provinces_and_indigenous_regions.csv

```diff
@@ -0,0 +1,13 @@
+id,name
+ocd-division/country:pa/province:bocas_del_toro,Bocas del Toro
+ocd-division/country:pa/province:chiriqui,Chiriquí
+ocd-division/country:pa/province:cocle,Coclé
+ocd-division/country:pa/province:colon,Colón
+ocd-division/country:pa/province:darien,Darién
+ocd-division/country:pa/province:herrera,Herrera
+ocd-division/country:pa/province:los_santos,Los Santos
+ocd-division/country:pa/province:panama,Panamá
+ocd-division/country:pa/province:panama_oeste,Panamá Oeste
+ocd-division/country:pa/province:veraguas,Veraguas
+ocd-division/country:pa/territory:kuna_yala,Kuna Yala
+ocd-division/country:pa/territory:ngabe-bugle,Ngäbe-Buglé

```

Comment:
Aren't there 4 territories total?

Reply:
Also, we would typically use the numbers for provinces and codes for territories from ISO 3166-2: https://en.wikipedia.org/wiki/ISO_3166-2:PA

Reply:
Hello, are you able to provide some examples?
Do I understand this correctly that I should replace:
"ocd-division/country:pa/**province:bocas_del_toro**/cp:1,Bocas del Toro" by "ocd-division/country:pa/**province:1**/cp:1,Bocas del Toro"
and "ocd-division/country:pa/**territory:ngabe-bugle**/cu:1,Ngäbe-Buglé-1" by "ocd-division/country:pa/**territory:nb**/cu:1,Ngäbe-Buglé-1"

Reply:
> Aren't there 4 territories total?

Yes, there are 4, but I've added only those which were attached in document from Government (for future election). I can add missing 2 (Emberá-Wounaan and Naso Tjër Di) if this is OK for you (both most likely won't have any ed for next election)

Reply:
I think trying to be complete for the entity type being added would be iedal so adding the other territories would be ideal